### PR TITLE
ci: add more owners for limited categories

### PR DESCRIPTION
**Pull request by jelbourn**
* Add alxhub, atscott, and AndrewKushnir to code owners
* Add atscott & AndrewKushnir to public-api and size-tracking
* Change public-api and size-tracking from requesting `-1` (all) to requesting four (still requiring three and two, respectively)

Follow-up to #37994

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -1155,13 +1155,15 @@ groups:
...
+        - AndrewKushnir
         - IgorMinar
         - alxhub
+        - atscott
         - jelbourn
         - petebacondarwin
         - pkozlowski-opensource
     reviews:
-      request: -1   # request reviews from everyone
+      request: 4   # request reviews from everyone
```
Comment (atscott) [inline .pullapprove.yml:1166]: This comment is no longer accurate.

File: .pullapprove.yml [hunk 2]
```diff
@@ -1155,13 +1155,15 @@ groups:
...
       required: 3   # require at least 3 approvals
```
Comment (atscott) [inline .pullapprove.yml:1167]: This comment seems a little redundant

File: .pullapprove.yml [hunk 3]
```diff
@@ -1179,13 +1181,15 @@ groups:
...
         - IgorMinar
         - alxhub
+        - atscott
         - jelbourn
         - petebacondarwin
         - pkozlowski-opensource
     reviews:
-      request: -1   # request reviews from everyone
+      request: 4   # request reviews from everyone
       required: 2   # require at least 2 approvals
```
Comment (atscott) [inline .pullapprove.yml:1193]:
```suggestion
      request: 4
      required: 2
```

**Review by atscott — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]
LGTM, thanks @jelbourn 👍 

I'm wondering if we can do similar thing for `circular-dependencies` group as well (request 3-4 and require 2 approvals)?

File: .pullapprove.yml (hunk 2 above)
Comment (josephperrott) [reply .pullapprove.yml:1167]: I would prefer to over comment throughout this file as we have found historically that its hard to remember/reason about what exactly every setting does and how/ordering PullApprove parses the file.

**Comment by jelbourn** [comment]
Comments addressed

File: .pullapprove.yml (hunk 2 above)
Comment (jelbourn) [reply .pullapprove.yml]: I added the comments back, though I do agree they're a bit redundant when the values are positive numbers

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request closed without merging**